examples/echo_app/components/statestore.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: dapr.io/v1alpha1
```

Comment:
add examples/../components to .gitignore

Reply:
added.